api/api/models/audio.py

```diff
@@ -192,7 +194,16 @@ class Audio(AudioFileMixin, AbstractMedia):
     alt_files = models.JSONField(
         blank=True,
         null=True,
-        help_text="JSON describing alternative files for this audio.",
+        help_text=d("""
+        JSON object containing information on alternative audio files. Each object
+        is expected to contain:
+
+        - `url`: URL reference to the file
+        - `filesize`: File size in bytes
+        - `filetype`: Extension of the file
+        - `bit_rate`: Bitrate of the file in bits/second
+        - `sample_rate`: Sample rate of the file in bits/second
+        """),
```

Comment:
It's a shame dedent is needed for `"""` strings.

I'm accustomed to seeing the use of implicit string concatenation in these fields (I think we do elsewhere), making the extra utility unnecessary (and therefore, not needing to remember or think about it). It also doesn't introduce the surprising newline after "Each object", which will render differently depending on context. Markdown will ignore it, but other contexts do not necessarily.

No real preference here either way, trade-offs in both cases to be sure.